scss/_settings_colors.scss

```diff
@@ -236,6 +240,7 @@ $colors--theme--text-inactive: var(--vf-color-text-inactive);
 
 $colors--theme--link-default: var(--vf-color-link-default);
 $colors--theme--link-visited: var(--vf-color-link-visited);
+$colors--theme--focus: var(--vf-color-focus);;
```

Comment:
I just have prettier set up on save in VS Code and never have to even think about it.

But it is also part of test script, so if you do `dotrun test` or `yarn test` it will run full linting.

Reply:
The one thing that `yarn test` won't catch is css generated into `import.css` when it shouldn't be. We have a [step that checks for that](https://github.com/canonical/vanilla-framework/blob/8f13d6f92189bfffcca216f3c364878aaea7f74e/.github/workflows/pr.yml#L23-L24) but it depends on creating a build first, which you might not expect from a linter.